docs/source/en/using-diffusers/marigold_usage.md

```diff
@@ -237,48 +324,46 @@ vis[0].save("einstein_normals.png")
   </div>
 </div>
 
-As can be seen, all areas with fine-grained structurers, such as hair, got more conservative and on average more correct predictions.
+As can be seen, all areas with fine-grained structurers, such as hair, got more conservative and on average more 
+correct predictions.
 Such a result is more suitable for precision-sensitive downstream tasks, such as 3D reconstruction.
 
 ## Quantitative Evaluation
```

Comment:
I would move this section closer towards the bottom since most users are probably more curious about learning how to use Marigold rather than evaluating it

Reply:
Done!